packages/frontity-org-theme/src/processors/image-frame.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { css, styled } from "frontity";
+import { Processor } from "@frontity/html2react/types";
+
+const Dot = styled.span`
+  display: inline-block;
+  width: 9px;
+  height: 9px;
+  margin-left: 5px;
+  border-radius: 100%;
+  background-color: rgba(15, 28, 100, 0.2);
+`;
+
+const Top = () => (
+  <div
+    css={css`
+      height: 32px;
+      padding-left: 10px;
+    `}
+  >
+    <Dot />
+    <Dot />
+    <Dot />
+  </div>
+);
+
+const imageFrame: Processor<React.HTMLProps<HTMLElement>> = {
+  name: "border-radius",
+  test: ({ node }) =>
+    node.type === "element" &&
+    node.props?.className?.split(" ").includes("has-browser-window"),
+
+  processor: ({ node }) => {
+    if (node.type !== "element") return node;
+
+    const topFrame: any = {
+      type: "element",
+      component: Top
+    };
+
+    node.children.unshift(topFrame);
+
+    node.props.css = css`
+      ${node.props.css};
+      box-shadow: 0 -1px 0 0 rgba(12, 17, 43, 0.04);
+      background-color: rgba(12, 17, 43, 0.02);
+      display: inline-flex;
+      flex-flow: column nowrap;
+
+      img {
+        border-bottom-left-radius: 11px;
+        border-bottom-right-radius: 11px;
+      }
```

Comment:
I think we should change this. First, the "browser-window" could be applied not only to `<img>` but also to `video`, `iframe` or any other element. Second, the border-radius should be handled by the parent `div` which could not have border-radius at all or have `has-border-radius-12px` and process by its own processor. This way it's more flexible and we could also add things like `has-shadow` for example.

If we do that, we may need to change some styles of the inner element like setting `margin: 0`.

Btw, I've changed the `/common-styles/` page to test it, feel free to modify it again.

Reply:
> First, the "browser-window" could be applied not only to <img> but also to video, iframe or any other element.
no problem, I've adjusted the CSS like:
```
      img,
      iframe,
      video {
        border-bottom-left-radius: 11px;
        border-bottom-right-radius: 11px;
      }
```
we can add whatever other element we want to target

> Second, the border-radius should be handled by the parent div which could not have border-radius at all or have has-border-radius-12px and process by its own processor. This way it's more flexible and we could also add things like has-shadow for example.
Oh, you're right that the div with the `has-browser-window` should also define it's `border-radius` but we also need a border radius on the image itself, because the border radius does not get "inherited" by the child `img` element`. So, we need to add the `border-bottom-left-radius` and `border-bottom-right-radius` for the `img` element as well.

With the styles that I've added the new example works as well.

Reply:
No no, I meant that we shouldn't need to add `border-radius` at all. That is managed by this other [processor](https://github.com/frontity/frontity.org/issues/5). We just have to add the class `has-border-radius-Xpx` to the parent `<div>`.

In order to achieve this I think that adding `overflow: hidden` to the `<div>` and `display: block` to the `<img>` it should be enough. I've tested it and it seems this way we could get rid of the `border-radius`. This way, it wouldn't be hardcoded and we would be more flexible.

Reply:
oh, I see! 🙈 

Yeah, that works great, you're doing my job for me 😅